Tools/DumpRenderTree/mac/DumpRenderTree.mm

```diff
@@ -133,7 +133,7 @@
 
 static RetainPtr<NSString> toNS(const std::string& string)
 {
-    return adoptNS([[NSString alloc] initWithUTF8String:string.c_str()]);
+    return adoptNS(@(string.c_str()]));
```

Comment:
I believe this will cause an overrelease. We should either keep it the way it currently is, or just `return @(string.c_str()])` and let it implicitly initialize a `RetainPtr`.